CommunicationLayer/PacketDecoder/test/TestPacketDecoder.cpp

```diff
@@ -125,6 +151,27 @@ void TestPacketDecoder::willReceiveCmuDataMessage()
    }
 }
 
+void TestPacketDecoder::willReceiveMpptDataMessage()
+{
+   QByteArray messageData = createValidMpptDataMessage();
```

Comment:
Mppt data was removed from valid messages being received. There is no use having any tests for it.